x-pack/plugin/core/src/main/resources/fleet-actions-results.json

```diff
@@ -0,0 +1,54 @@
+{
+  "index_patterns": [
+    ".fleet-actions-results"
+  ],
+  "data_stream": {},
+  "template": {
+    "settings": {
+      "index.lifecycle.name": ".fleet-actions-results-ilm-policy"
+    },
+    "mappings": {
+      "_meta": {
+        "version": "${fleet.version}"
+      },
+      "dynamic": false,
+      "properties": {
+        "action_id": {
+          "type": "keyword"
+        },
+        "agent_id": {
+          "type": "keyword"
+        },
+        "action_data": {
+          "enabled": false,
+          "type": "object"
+        },
+        "data": {
+          "enabled": false,
+          "type": "object"
+        },
+        "error": {
+          "type": "text",
+          "fields": {
+            "keyword": {
+              "type": "keyword",
+              "ignore_above": 256
```

Comment:
nit: can we increase it to 1024, seems like it's often a default with other integrations datastreams?